connman/unit/test-clat.c

```diff
@@ -103,6 +107,14 @@ int connman_inet_ifindex(const char *name)
 			return 3;
 	}
 
+	if (g_str_has_prefix(name, "vpn")) {
+		int index = (int)name[strlen(name) - 1] + 10;
```

Comment:
nit: cast seems unnecessary, char should auto-promote to int, or?

Reply:
I guess this is not needed, but it is only in tests.